lightwood/mixers/sklearn_mixer.py

```diff
@@ -0,0 +1,185 @@
+                    model = model_class(**model_kwargs)
+                    model.fit(X_train, Y_train, sample_weight=sample_weight)
+                    accuracy = balanced_accuracy_score(
+                        Y_test,
+                        model.predict(X_test)
+                    )
+
+                    model_classes_and_accuracies.append((
+                        (model_class, model_kwargs),
+                        accuracy
+                    ))
+                
+                (best_model_class, best_model_kwargs), _ = max(
+                    model_classes_and_accuracies,
+                    key=lambda x: x[-1]
+                )
+
+                self.targets[target_col_name]['model'] = best_model_class(**best_model_kwargs)
+
+            elif self.targets[target_col_name]['type'] == COLUMN_DATA_TYPES.NUMERIC:
+                model_classes_and_accuracies = []
+                for model_class, model_kwargs in REGRESSION_MODELS:
+                    model = model_class(**model_kwargs)
+                    model.fit(X_train, Y_train)
+                    accuracy = r2_score(Y_test, model.predict(X_test))
+
+                    log.info('[MODEL/COLUMN/TEST_ACCURACY]: {}/{}/{}'.format(
+                        model_class.__name__,
+                        target_col_name,
+                        accuracy
+                    ))
+
+                    model_classes_and_accuracies.append((
+                        (model_class, model_kwargs),
+                        accuracy
+                    ))
+                
+                (best_model_class, best_model_kwargs), best_accuracy = max(
+                    model_classes_and_accuracies,
+                    key=lambda x: x[-1]
+                )
+
+                self.targets[target_col_name]['model'] = best_model_class(**best_model_kwargs)
+
+                log.info('Selecting {} for column {} (accuracy={})'.format(
+                    best_model_class.__name__,
+                    target_col_name,
+                    best_accuracy
+                ))
+
+                if self.quantiles is not None:
+                    self.targets[target_col_name]['quantile_models'] = {}
+                    for i, quantile in enumerate(self.quantiles):
+                        self.targets[target_col_name]['quantile_models'][i] = best_model_class(**best_model_kwargs)
+
+            else:
+                self.targets[target_col_name]['model'] = None
+        
+        # Fit best model of each column on [train_ds + test_ds]
+        for target_col_name in self.targets:
+            Y_train = train_ds.get_column_original_data(target_col_name)
+            Y_test = test_ds.get_column_original_data(target_col_name)
+            X, Y = (X_train + X_test), (Y_train + Y_test)
+
+            if self.targets[target_col_name]['model'] is not None:
+                self.targets[target_col_name]['model'].fit(X, Y)
+
+            if 'quantile_models' in self.targets[target_col_name]:
```

Comment:
Is Mindsdb_native currently setting `quantile_models` ? Or is this a feature to be implemented later ?

Reply:
it's set here: https://github.com/mindsdb/lightwood/blob/ff4e9edb3df097ce2247359d2a4d10586a5d356f/lightwood/mixers/sklearn_mixer.py#L132